internal/pkg/agent/install/.flavors

```diff
@@ -0,0 +1,13 @@
+basic:
```

Comment:
This is duplicating https://github.com/elastic/elastic-agent/blob/main/_meta/.flavors